### ISSUE TITLE
Multiline text is area is displayed as single string

### ISSUE THREAD

**Issue by iArnold** [red/red#1777]
Run this code in GUI-console
multi-line-text: {
Hi,

This is

multiline

text in area
}

view [
	multi-line-area: area 200x200
	return
	button "fill area" [multi-line-area/text: multi-line-text ]
	button "empty area" [multi-line-area/text: ""]
]
![2016-04-01_142134](https://cloud.githubusercontent.com/assets/1219809/14206713/4ef9066c-f815-11e5-8782-797c4860a731.png)

**Comment by meijeru** [comment]
This could be related to #1488

**Comment by iArnold** [comment]
If I Cut&Paste this text from the area and put it into a code editor then I see CR LF markers.

**Comment by meijeru** [comment]
That itself is not conclusive, since the Cut&Paste could do that !?!

**Comment by iArnold** [comment]
Perhaps, I have not noticed that before. 
I have now a script that fills such area/text and that works fine when run from GUI-console but when I have compiled version the area/text remains empty. Unfortunately I cannot access the source now.

**Comment by WiseGenius** [comment]
@iArnold What happens when you cut that text back from the code editor and paste it back into the `area`?
For me on Windows, this part of your code gives me all `^/`s:
```
red>> multi-line-text: {
{    Hi,
{    
{    This is
{    
{    multiline
{    
{    text in area
{    }
== {^/Hi,^/This is^/multiline^/text in area^/}
```
#1488 then explains why this is displayed as one line in the `area`.
When I cut the text from the `area` and paste it into a code editor, some code editors add the missing `^M` as expected by Windows.
So when I cut the text back from the editor and paste it back into the `area`, it is displayed as multiline.

For me, this code runs as expected when both interpreted and compiled:
```
Red [
	needs: 'View
]

multi-line-text: {^M^/Hi,^M^/This is^M^/multiline^M^/text in area^M^/}

view [
	multi-line-area: area 200x200
	return
	button "fill area" [multi-line-area/text: multi-line-text ]
	button "empty area" [multi-line-area/text: ""]
]
```

**Comment by iArnold** [comment]
Compiled (left) versus Interpreted (right)
![2016-04-04_093209](https://cloud.githubusercontent.com/assets/1219809/14241060/ce4bfa10-fa49-11e5-9cc0-f143aa283eba.png)

**Comment by iArnold** [comment]
@WiseGenius  When I cut and past directly the right situation stays the same. When I cut and paste to an editor the lines show up and cutting and pasting that I end up with the left.

**Comment by WiseGenius** [comment]
@iArnold, Yes. The cutting and pasting situation is what I'd currently expect.

I also reproduced the difference in behaviour between compiling and interpreting your original code. Here's the reason:

When the following code is in a file with Windows (CR LF) endings:
```
Red []

s: {a
b}

probe s
```
The interpreter gives `"a^/b"` while the compiled executable gives `"a^M^/b"`.

@dockimbel, Which is the correct result?

@iArnold, As for how each result is displayed in the `area`, that's #1488.

**Comment by dockimbel** [comment]
@WiseGenius `"a^/b"` is the correct result, LF is used for delimiting new-lines in Red (same as Rebol).

**Comment by dockimbel** [comment]
Fixed by commit [a39c3edb](https://github.com/red/red/commit/dd4009baaab2ed4933dcc23e7813a4f1a39c3edb).
